packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -21,8 +21,9 @@
 								<tera-operator-port-icon v-if="input.type" :portType="input.type" />
 							</template>
 						</Chip>
-					</div>
+					</aside>
 					<tera-output-dropdown
+						class="ml-auto"
```

Comment:
what does `ml-auto` and what kind of tests are needed to verify that the embedded operators are okay?

Reply:
* `ml-auto` is `margin-left: auto` via _Tailwind CSS_.
* To test the operator you can just open an operator of each type mentioned above.